src/common/string_util.h

```diff
@@ -90,22 +113,26 @@ inline std::optional<bool> FromChars(const std::string_view& str)
 
   return std::nullopt;
 }
-
+#if 0
```

Comment:
If this is replaced by the above implementation, can just remove it rather than `#if 0`'ing it out.